simple_functions.py

```diff
@@ -1,7 +1,9 @@
 # Custom python functions
 
 def double_number(a):
+'''This function takes the input and prints the number before and after doubling it and returns the doubled number'''
```

Comment:
Elaborate more regarding the parameter taken by the function. Please refer to pandas docstring.